docs/src/operate/c8y/software-management.md

```diff
@@ -112,6 +112,20 @@ If you are upgrading %%te%% from any version prior to `1.0.0` (including `1.0.0-
 
 ## Configuration {#configuration}
 
+### Switching to Advanced Software Management
+
+%%te%% supports both [Legacy Software Management](https://cumulocity.com/docs/device-integration/fragment-library/#legacy-software-management) and [Advanced Software Management](https://cumulocity.com/docs/device-integration/fragment-library/#advanced-software-management) features of Cumulocity IoT.
+By default, Legacy Software Management is selected.
+To switch to Advanced Software Management, use the `tedge` command to change the following configurations:
```

Comment:
We should probably also mention that the mapper needs to be restarted before the setting takes effect.

Reply:
Good point. Added.